.dockerignore

```diff
@@ -0,0 +1,10 @@
+vendor/pkg
+vendor/src
+pkg
+hooks
+bin
+
+# Other stuff we just don't need
+.gitignore
+.git
+.travis.yml
```

Comment:
iirc the .dockerignore file is only important for not having to hand a lot of data to the docker daemon over http. I agree that .git and some of the directories up there should definitely be in there, but ignoring .gitignore and .travis.yml seems a little random

Reply:
They were picked randomly, so I'll remove them